cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -0,0 +1,131 @@
+{% extends "templates/base_page.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/text-indent/_macro.njk" import onsTextIndent %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+
+
+{% block header_area %}
+    <div class="analysis-header {% if page.updates %}analysis-header--flush{% endif %}">
+        <div class="ons-container">
+            {% include "templates/components/navigation/breadcrumbs.html" %}
+            <p class="analysis-header__doc-type ons-u-fs-l">{{ _("Methodology") }}</p>
+
+            <div class="ons-grid ons-grid-flex-gap">
+                <div class="ons-grid__col ons-col-10@m ">
+                    <h1 class="ons-u-fs-3xl analysis-header__heading">
+                        {{ page.title }}
+                    </h1>
+
+                    <div class="analysis-header__summary">{{ page.summary|richtext() }}</div>
+                </div>
+            </div>
+        </div>
+
+        {% block release_meta %}
+            <div class="ons-container analysis-header__releases">
+                <div>
+                    <span class="ons-u-fs-r--b analysis-header__releases-label">{{ _("Published") }}:</span>
+                    <span class="ons-u-nowrap">{{ page.published_date|date("DATE_FORMAT") }}</span>
+                </div>
+                <div>
+                    <span class="ons-u-fs-r--b analysis-header__releases-label">{{ _("Last revised:") }}:</span>
```

Comment:
```suggestion
                    <span class="ons-u-fs-r--b analysis-header__releases-label">{{ _("Last revised") }}:</span>
```

Reply:
Done as part of [7bad8a3](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/7bad8a3ecef388d5ea915f5db34c56fefc4f949a)